modules/ztp-image-based-upgrade-installing-lca.adoc

```diff
@@ -19,15 +19,21 @@ Install the {lcao} with {ztp-first} to do an image-based upgrade.
 .Example `LcaSubscriptionNS.yaml` file
 [source,yaml]
 ----
-apiVersion: v1
-kind: Namespace
+apiVersion: operators.coreos.com/v1
```

Comment:
@achuzhoy Please check if this is correct as the copypasted YAML had `apiVersion: operators.coreos.com/v1alpha1`. If I'm not mistaken, this was updated for the GA release of LCA to `apiVersion: operators.coreos.com/v1`

Reply:
If I'l looking at an example here: https://docs.okd.io/4.16/operators/admin/olm-adding-operators-to-cluster.html

It's v1alpha1


Also,if you execute the following against any cluster `oc api-resources|grep "Subscription"|grep oper`

You'll see v1alpha1